client/mysqlbinlog.cc

```diff
@@ -821,8 +820,7 @@ static bool print_row_event(PRINT_EVENT_INFO *print_event_info, Log_event *ev,
   if (is_stmt_end && !result)
   {
     if (print_event_info->print_row_count)
-      fprintf(result_file, "# Number of rows: %s\n",
-              llstr(print_event_info->row_events, ll_buff));
+      fprintf(result_file, "# Number of rows: %lld\n", print_event_info->row_events);
```

Comment:
🤔 Turning on `-Wconversion` (MDEV-26896) will complain about cases like these before this replacement.
🧐 It will also complain about `char *ullstr(longlong value,char *buff)` because it delegates a `ulonglong` arg to `longlong10_to_str(value,buff,10);` which is sign-agnostic.